### PR TITLE
Changed date from 27 to 28 and Clifton's bio fixes

### PR DESCRIPTION
**Pull request by ckiss**
Just changed the data. tito link still needs to be changed.

### CODE REVIEW

**Comment by ckiss** [comment]
I changed the tickets link too.

File: 2015-condenast/index.html [hunk 1]
```diff
@@ -1084,9 +1084,15 @@ <h3 class="title">Clifton Cunningham</h3>
                 </p>
 
                 <p class="bio">
-                  <b>About Clifton</b> Clifton is the CTO at TES Global, a 100 year old educational publisher that is busy transforming itself into the largest online social network for teachers, providing tools and services that empower teachers to be better teachers across the globe.<br>
-                  He was previously the CTO at the MailOnline, one of the worlds largest online news sites, and helped to transform the entire technology platform, and culture, from a legacy monolithic java stack surrounded by fear into a fast moving micro service based platform built on top of NodeJS and Clojure and built on the foundations of programmer anarchy and freedom from fear.<br>
-                  Clifton still writes code, starting his journey with NodeJS by writing an open source CMS called Calipso a few years ago, as well as contributing to projects internally at TES whenever he gets the opportunity.
+                  Clifton is the CTO at TES Global, a 100 year old educational publisher that is
```
Comment (totherik) [inline 2015-condenast/index.html]: No `<b>About Clifton</b>`?

**Comment by ckiss** [comment]
Hi @totherik. Fixed. Thanks. You have a good eye :)

**Comment by totherik** [comment]
Perfect! Thanks!

**Pull request merged**